src/rgw/rgw_sal_motr.cc

```diff
@@ -2719,13 +2694,16 @@ int MotrObject::fetch_null_obj_reference(const DoutPrefixProvider *dpp, std::str
   if (this->store->get_obj_meta_cache()->get(dpp, null_ref_key, bl)) {
     rc = this->store->do_idx_op_by_name(bucket_index_iname,
                               M0_IC_GET, null_ref_key, bl);
-    ldpp_dout(dpp, 20) <<__func__<< ": GET null index entry, rc = "<< rc << dendl;
-    if (rc < 0)
-    {
-      ldpp_dout(dpp, 0) << __func__ << "ERROR: Key- "<< null_ref_key <<" does not exist. rc=" << rc << dendl;
+    ldpp_dout(dpp, 20) <<__func__<< ":  GET null index entry, rc = "<< rc << dendl;
```

Comment:
not sure why null_ref_key removed from log, it should help while debugging i believe

Reply:
Not removed, we have moved this log to line -2713.